jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileContextProvider.kt

```diff
@@ -198,13 +201,7 @@ class DefaultCodeWhispererFileContextProvider(private val project: Project) : Fi
         return chunks.take(CodeWhispererConstants.CrossFile.CHUNK_SIZE)
     }
 
-    override fun isTestFile(psiFile: PsiFile): Boolean {
-        val path = runReadAction { contentRootPathProvider.getPathToElement(project, psiFile.virtualFile, null) ?: psiFile.virtualFile.path }
-        return TestSourcesFilter.isTestSources(psiFile.virtualFile, project) ||
-            path.contains("""test/""") ||
-            path.contains("""tst/""") ||
-            path.contains("""tests/""")
-    }
```

Comment:
moved to `CodeWhispererFileCrawler` and previously we had duplicate code in terms of functionality


https://github.com/aws/aws-toolkit-jetbrains/pull/3774/files#diff-faff4822410665507967aaa15bfe64abab03286e73a1bdc9c9d5046408de467bR78